src/libs/ReportUtils.ts

```diff
@@ -1075,6 +1075,13 @@ function findLastAccessedReport(
     return adminReport ?? sortedReports.at(-1) ?? null;
 }
 
+/**
+ * Whether the provided report is a closed expense report with no expenses
+ */
+function isClosedExpenseReportWithNoExpenses(report: OnyxEntry<Report>): boolean {
+    return report?.statusNum === CONST.REPORT.STATUS_NUM.CLOSED && report?.stateNum === CONST.REPORT.STATE_NUM.OPEN;
```

Comment:
Apologies since it looks like we missed this during the proposal phase. But we cannot/should not rely on the `state` of the report being `Open`. I think in most (all ?) cases it will be `Approved`.

The criteria for displaying this UI should be:
1. `status` of `closed` (`report?.statusNum === CONST.REPORT.STATUS_NUM.CLOSED`)
2. type: `expense`
3.  0 expenses on the report

Reply:
@rayane-djouah Could you kindly check on this and update?